### PR TITLE
Allow running tidy on single files

### PR DESCRIPTION
**Pull request by foursixnine**
As a follow up to 3b277957a and os-autoinst/openQA/4216 add more
information to the xml metadata to avoid guessing where a remote worker
is running

### CODE REVIEW

File: tools/tidy [hunk 1]
```diff
@@ -69,6 +75,11 @@ fi
 
 find-files() {
     local files=()
+
+    if [[ -f $filename ]]; then
+        ls $filename
```
Comment (okurz) [inline tools/tidy:80]: do you need to list the file instead of just `echo $filename`?

Comment (foursixnine) [reply tools/tidy:80]: I’m a bit paranoid, either way should be ok.

Comment (Martchus) [reply tools/tidy:80]: The parameter should be quoted and considering the `if` it makes more sense to use `echo`.

Comment (okurz) [reply tools/tidy:80]: Actually I think the whole function `find_files` should only be called if no file parameter is supplied, e.g. change line R102 

```diff
- find-files | xargs perltidy --pro=.../.perltidyrc
+ [[ -f $filename ]] && perltidy --pro=.../.perltidyrc $filename || find-files | xargs perltidy --pro=.../.perltidyrc

**Review by okurz — CHANGES_REQUESTED** [review]

File: tools/tidy (hunk 1 above)
Comment (foursixnine) [reply tools/tidy]: @okurz I like that! done.

**Review by okurz — APPROVED** [review]

**Comment by okurz** [comment]
It's ok

**Pull request merged**